deployments/terraform/modules/sqs/main.tf

```diff
@@ -38,9 +39,10 @@ module "dead" {
 
   messageRetentionSeconds = var.messageRetentionSeconds
 
-  alarm_create              = var.alarm_dead_create
-  alarm_period              = var.alarm_dead_period
-  alarm_threshold           = var.alarm_dead_threshold
-  alarm_evaluation_periods  = var.alarm_dead_evaluation_periods
-  alarm_datapoints_to_alarm = var.alarm_dead_datapoints_to_alarm
+  alarm_create                    = var.alarm_dead_create
+  alarm_period                    = var.alarm_dead_period
+  alarm_visible_message_threshold = var.alarm_dead_threshold
+  alarm_message_age_threshold     = var.alarm_dead_message_age_threshold
```

Comment:
for the dead letter queue we don't need an age based alarm..  
message in the DLQ will always age as they are not consumed on a regular basis. it's more of a backup of failed messages.